src/app/public/modules/text-expand-repeater/text-expand-repeater.component.ts

```diff
@@ -40,15 +40,25 @@ let nextId = 0;
   ]
 })
 export class SkyTextExpandRepeaterComponent implements AfterViewInit, OnChanges {
-  @Input()
-  public maxItems: number;
 
+  /**
+   * Specifies the data to truncate.
```

Comment:
In the current docs, this is marked as required. Should we add `@required` here?

Reply:
Not sure why it was marked as required in the first place. You can still load the component without this value and there are no errors. Obviously, it doesn't do much with out any data to truncate, but there are no errors and its technically not required. The same goes for the other properties you marked below.